erpnext/stock/doctype/pick_list/pick_list.py

```diff
@@ -47,11 +48,24 @@ def validate_delivery_date(self):
 		self.delivery_date = min(order_delivery_dates)
 
 	def validate_stock_qty(self):
+		"""User should not allowed to create pick list if sales order item qty exceed"""
+		pick_list_items = frappe.get_all("Pick List Item", fields=["*"])
 		for item in self.locations:
 			if item.get("sales_order_item"):
+				pick_list_qty = 0
+				item_sales_order_qty = frappe.db.get_value("Sales Order Item", item.get("sales_order_item"), "qty")
```

Comment:
If there is no sales order in the row, this'll fail at row 63

Reply:
line 54 has check for sales order item